wallet/seed.go

```diff
@@ -48,6 +48,28 @@ func KeyFromPhrase(phrase string) (types.PrivateKey, error) {
 	return key, nil
 }
 
+func SeedFromPhrase(seed *[32]byte, phrase string) error {
```

Comment:
reminder: remove this after switching to `KeyFromPhrase`

Reply:
Done!!